packages/stores/types.ts

```diff
@@ -340,6 +359,11 @@ type TTradersHubStore = {
     setSelectedAccount: (account: { login?: string; account_id?: string }) => void;
     toggleAccountTransferModal: () => void;
     is_demo: boolean;
+    is_account_type_modal_visible: boolean;
+    toggleAccountTypeModalVisibility: () => void;
+    account_type_card: string;
+    selectAccountTypeCard: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
```suggestion
    selectAccountTypeCard: (value: string) => void;
```

@thisyahlen-deriv Never trust the existing types we have :)) Always double check :D

Reply:
Will do 🙏